src/metabase/models.clj

```diff
@@ -54,9 +56,12 @@
    [metabase.models.view-log :as view-log]
    [metabase.plugins.classloader :as classloader]
    [metabase.public-settings.premium-features :refer [defenterprise]]
+   [metabase.search :as search]
+   [metabase.search.postgres.ingestion :as search.ingestion]
```

Comment:
well it should be something like `search/update-index!`, and then it'll look at what's the current engine and proceed, right?